var/spack/repos/builtin/packages/bedtools2/package.py

```diff
@@ -22,11 +25,53 @@ class Bedtools2(Package):
     version("2.26.0", sha256="15db784f60a11b104ccbc9f440282e5780e0522b8d55d359a8318a6b61897977")
     version("2.25.0", sha256="159122afb9978015f7ec85d7b17739b01415a5738086b20a48147eeefcf08cfb")
     version("2.23.0", sha256="9dacaa561d11ce9835d1d51e5aeb092bcbe117b7119663ec9a671abac6a36056")
+    version("2.21.0", sha256="51380b718cc5f2023d0738374a86a7d42fcca2385cf051a6f0c95d9c2624d78d")
 
     depends_on("zlib")
     depends_on("bzip2", when="@2.29:")
     depends_on("xz", when="@2.29:")
     depends_on("python", type="build")
+    depends_on("samtools", type="test")
+
+    conflicts(
+        "%gcc@11:",
+        when="@:2.26.0",
+        msg="bedtools2 2.26.0 and earlier requires GCC 10 or earlier to build",
+    )
 
+    # https://github.com/arq5x/bedtools2/commit/58e9973af1b3f5e3b26e5584aad7dc7b720f8765
+    patch("fix-binary-variable.patch", when="@:2.29.2")
```

Comment:
Is it possible to replace this with a url to the patch, like this?
```suggestion
    # https://github.com/arq5x/bedtools2/commit/58e9973af1b3f5e3b26e5584aad7dc7b720f8765
    patch(
        "https://github.com/arq5x/bedtools2/commit/58e9973af1b3f5e3b26e5584aad7dc7b720f8765.patch?full_index=1",
        sha256="03ddfe3a8185f3a2ac40bcaf5e442bfc290a311139869456b645a4c36b148d56",
        when="@:2.29.2",
    )
```
The benefits are: no copy of code (license implications), traceability, and smaller repository here.